2018-05-Berlin/attendee-rsvp.md

```diff
@@ -0,0 +1,81 @@
+# Node.js Collaborator Summit, 2018-05-31 - 2018-06-01, Berlin, Germany
+This document serves as an RSVP list for attendees in person, attendees remote, and the collaborator's dinner hosted by the Node Foundation.
+
+This is how we will track amount of food and seats. Submit a PR if you need to RSVP or update your RSVP.
+
+### Attendees In-Person 
+If you will be in Berlin to attend the Collaborator's Summit - add your name.
+
+1. [Myles Borins](https://github.com/MylesBorins) // Organizer
```

Comment:
can we change this to an unordered list

Reply:
i think having an ordered list makes it easier to see how many people are attending, no?

Reply:
but there's no inherent order to this list, and making it ordered might cause people to infer things that shouldn't be inferred. no other attendance lists that i know of in the org are number ordered. usually we do alphabetical.

Reply:
Alphabetical by user name seems reasonable to me.

If we want to have a separate list for organizers that makes sense.

Reply:
+1 for alphabetical, also helps if you're quickly scanning through to check if a certain person is attending.

Reply:
Looks like @chowdhurian is missing.